File: backend/src/main/java/um/edu/ar/backend/infrastructure/web/dto/EventoResponse.java
package um.edu.ar.backend.infrastructure.web.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class EventoResponse {
    private Long id;
    private String nombre;
    private String resumen;
    private String descripcion;
    private LocalDateTime fechaHora;
    private String direccion;
    private String tipoNombre;
    private Integer filas;
    private Integer columnas;
    private Double precio;
}

File: cliente_movil/composeApp/src/commonMain/kotlin/um/edu/ar/model/EventoResponse.kt
package um.edu.ar.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class EventoResponse(
    val id: Long,
    val nombre: String,
    val resumen: String,
    val descripcion: String,
    @SerialName("fechaHora")
    val fechaHora: String,
    val direccion: String,
    val tipoNombre: String,
    val filas: Int,
    val columnas: Int,
    @SerialName("precio")
    val precio: Double
)
